platform/src/main/resources/config/anchor-config-default-values.yaml

```diff
@@ -330,61 +330,92 @@ metrics:
 #########################
 ## The events being sent from the platform are the ones described in the `Events Schema.yml` file
 events:
-
   ## @param: enabled
   ## @type: bool
   ## Whether to enable events.
   #
   enabled: false
 
+  ## @param: event_type_to_queue
+  ## @type: map
+  ## Mapping of the event type to the queue name that messages are published to
+  event_type_to_queue:
```

Comment:
Could you plz run it locally with `example.anchor-config.yaml` to make sure that config still works?

Reply:
The `events:` is updated as the following. 
```yaml
events:
  enabled: false
  event_type_to_queue:
    quote_created: ap_event_quote_created
    transaction_created: ap_event_transaction_created
    transaction_status_changed: ap_event_transaction_status_changed
    transaction_error: ap_event_transaction_error
  publisher:
    type: kafka
    kafka:
      bootstrap_servers: localhost:29092
      client_id:
      retries: 1
      linger_ms: 1000
      batch_size: 10
```